specs/_features/maxeb_increase/capella.py

```diff
@@ -3589,11 +3623,12 @@ def get_expected_withdrawals(state: BeaconState) -> Sequence[Withdrawal]:
             ))
             withdrawal_index += WithdrawalIndex(1)
         elif is_partially_withdrawable_validator(validator, balance):
+            excess = get_validator_excess_balance(validator, balance)
             withdrawals.append(Withdrawal(
                 index=withdrawal_index,
                 validator_index=validator_index,
                 address=ExecutionAddress(validator.withdrawal_credentials[12:]),
-                amount=balance - MAX_EFFECTIVE_BALANCE,
+                amount=balance - excess,
```

Comment:
should this be `excess` (instead of `balance - excess`)?

Reply:
consider just doing `amount=get_validator_excess_balance(validator, balance)`

Reply:
nice catch! done :-)